installing/installing_aws/installing-restricted-networks-aws.adoc

```diff
@@ -98,8 +98,13 @@ include::modules/installation-configure-proxy.adoc[leveloffset=+2]
 
 //include::modules/installation-three-node-cluster.adoc[leveloffset=+2]
 
+// Creating the Kubernetes manifest and Ignition config files
 include::modules/installation-user-infra-generate-k8s-manifest-ignition.adoc[leveloffset=+2]
 
+[role="_additional-resources"]
+.Additional resources
+* xref:../../installing/installing_aws/installing-restricted-networks-aws-installer-provisioned.adoc#manually-create-iam_installing-restricted-networks-aws-installer-provisioned[Manually creating long-term credentials]
```

Comment:
The 4.13 cherry pick seems to be barfing on this reference.

Reply:
@dfitzmau, will you PTAL? You're probably going to need to open manual cherry-picks to address this issue.

Reply:
Yes, of course. 

I raised:

- https://github.com/openshift/openshift-docs/pull/66072/files
- https://github.com/openshift/openshift-docs/pull/66073/files